src/openpgp.js

```diff
@@ -374,6 +374,12 @@ function checkCleartextMessage(message) {
  * Format user ids for internal use.
  */
 function formatUserIds(options) {
+  if (!options.curve) {
```

Comment:
This is a work around because some tests do not expect extra parameters like 'curve' and 'material'.